addon/components/sortable-group.js

```diff
@@ -111,7 +118,11 @@ export default Component.extend({
         items.invoke('thaw');
       });
     });
-
-    this.sendAction('onChange', models);
+    
+    if (groupModel === null) {
```

Comment:
Well, I'm assuming that if the object passed as `model` is null, there shouldn't be any items in the object for it to invoke `commit()`.  

Otherwise, we'd need to pre-define `model` as something other than `null`, because that's what it's using to check if user has passed in something `undefined`.